bindings/pydeck/pydeck/settings.py

```diff
@@ -26,9 +26,12 @@ class Settings:
         ```
     """
 
-    def __init__(self, custom_libraries=None):
+    def __init__(self, custom_libraries: list = None):
```

Comment:
`list` with a lower case `l` is intentional? https://stackoverflow.com/q/39458193